content/blog/2019/12/2019-12-12-board-election-results.adoc

```diff
@@ -0,0 +1,95 @@
+---
+layout: post
+title: "2019 Jenkins Board and Officer Elections Results"
+tags:
+- community
+- governance
+- governance-board
+- elections
+authors:
+- oleg_nenashev
+opengraph:
+  image: /images/post-images/elections-2019/election-2019-opengraph.png
+---
+
+image:/images/post-images/elections-2019/election-2019-opengraph.png[Jenkins Elections, role=center, float=right, height=178]
+
+The Jenkins community is conducting the 2019 elections for Board and Officer positions.
+The call for nominations has concluded on Nov 25, and the election results have been link:https://groups.google.com/forum/#!topic/jenkinsci-dev/bX2qhzoEWXo[announced in the developer mailing list] on November 28.
+
+On behalf of the Jenkins community, we congratulate all elected board members and officers!
+We also thank Jenkins contributors participated in elections this year: all nominees and hundreds of voters.
+These are the first elections ever conducted by the Jenkins project, and it is a big milestone for Jenkins as a project.
+
+Election results:
+
+* link:https://github.com/uhafner[Ullrich Hafner], link:https://github.com/slide[Alex Earl] and link:https://github.com/oleg-nenashev[Oleg Nenashev] will join link:https://github.com/kohsuke[Kohsuke Kawaguchi] and link:https://github.com/rtyler/[R. Tyler Croy] in the link:/project/governance/#governance-board[Jenkins Governance Board]
+* link:https://github.com/daniel-beck[Daniel Beck] was elected as a link:/project/team-leads/#security[security officer]
+* link:https://github.com/alyssat[Alyssa Tong] was elected an link:/project/team-leads/#events[events officer]
+* Nominations for Infrastructure, Release and Documentation officer positions were uncontested.
+* link:https://github.com/olblak[Olivier Vernin] was elected as an link:/project/team-leads/#infrastructure[infrastructure officer] (uncontested)
+* link:https://github.com/ogondza[Oliver Gondza] was elected as an link:/project/team-leads/#release[release officer] (uncontested)
+* link:https://github.com/MarkEWaite[Mark Waite] was elected as an link:/project/team-leads/#documentation[documentation officer] (uncontested)
```

Comment:
It would be "a" instead of "an" since "documentation" does not start with a vowel sound.